docs/nitpick-exceptions

```diff
@@ -1,2 +1,7 @@
 # photutils.morphology
 py:class photutils.morphology.CompoundModel1
+
+# photutils.detection
+py:class photutils.detection.findstars.StarFinderBase
```

Comment:
Yes, please add them to the public classes.  I already do this in `photutils`, e.g. the base background classes:
http://photutils.readthedocs.io/en/latest/api/photutils.background.MMMBackground.html#photutils.background.MMMBackground